core/tests/protractor-mobile.conf.js

```diff
@@ -0,0 +1,506 @@
+ */
+
+require('dotenv').config();
+var FirebaseAdmin = require('firebase-admin');
+var HtmlScreenshotReporter = require('protractor-jasmine2-screenshot-reporter');
+var glob = require('glob');
+var path = require('path');
+var fs = require('fs');
+var childProcess = require('child_process');
+var Constants = require('./protractor_utils/ProtractorConstants');
+var DOWNLOAD_PATH = path.resolve(__dirname, Constants.DOWNLOAD_PATH);
+var exitCode = 0;
+
+var suites = {
+    // The tests on Travis are run individually to parallelize
+    // them. Therefore, we mention the complete directory
+    // in 'full'.
+    full: [
+      'protractor/*.js',
+      'protractor_desktop/*.js'
+    ],
+
+    // Unfortunately, adding more than one file to a test suite results in
+    // severe instability as of Chromedriver 2.38 (Chrome 66).
+    accessibility: [
+      'protractor/accessibility.js'
+    ],
+
+    additionalEditorFeatures: [
+      'protractor_desktop/additionalEditorFeatures.js'
+    ],
+
+    additionalEditorFeaturesModals: [
+      'protractor_desktop/additionalEditorFeaturesModals.js'
+    ],
+
+
+    additionalPlayerFeatures: [
+      'protractor_desktop/additionalPlayerFeatures.js'
+    ],
+
+    adminPage: [
+      'protractor_desktop/adminTabFeatures.js'
+    ],
+
+    blogDashboard: [
+      'protractor_desktop/blogDashboard.js'
+    ],
+
+    classroomPage: [
+      'protractor_desktop/classroomPage.js'
+    ],
+
+    classroomPageFileUploadFeatures: [
+      'protractor_desktop/classroomPageFileUploadFeatures.js'
+    ],
+
+    collections: [
+      'protractor_desktop/collections.js'
+    ],
+
+    contributorDashboard: [
+      'protractor_desktop/contributorDashboard.js'
+    ],
+
+    coreEditorAndPlayerFeatures: [
+      'protractor_desktop/coreEditorAndPlayerFeatures.js'
+    ],
+
+    creatorDashboard: [
+      'protractor_desktop/creatorDashboard.js'
+    ],
+
+    embedding: [
+      'protractor_desktop/embedding.js'
+    ],
+
+    explorationImprovementsTab: [
+      'protractor_desktop/explorationImprovementsTab.js'
+    ],
+
+    explorationFeedbackTab: [
+      'protractor_desktop/explorationFeedbackTab.js'
+    ],
+
+    explorationHistoryTab: [
+      'protractor_desktop/explorationHistoryTab.js'
+    ],
+
+    explorationStatisticsTab: [
+      'protractor_desktop/explorationStatisticsTab.js'
+    ],
+
+    explorationTranslationTab: [
+      'protractor_desktop/explorationTranslationTab.js'
+    ],
+
+    extensions: [
+      'protractor_desktop/extensions.js'
+    ],
+
+    featureGating: [
+      'protractor/featureGatingFlow.js'
+    ],
+
+    fileUploadFeatures: [
+      'protractor_desktop/voiceoverUploadFeatures.js'
+    ],
+
+    fileUploadExtensions: [
+      'protractor_desktop/fileUploadExtensions.js'
+    ],
+
+    learnerDashboard: [
+      'protractor_desktop/learnerDashboard.js'
+    ],
+
+    learner: [
+      'protractor/learnerFlow.js'
+    ],
+
+    library: [
+      'protractor/libraryFlow.js'
```

Comment:
TBH I'm not sure, should I test which of them would run on the mobile viewport?

Reply:
> Thanks @Lawful2002 for the PR. I have taken a full pass and left a few comments, PTAL.
> 
> A general comment from my side would be can we not use only the `protractor.conf.js` configuration file to handle the mobile tests too because I think you are only updating the window size for mobile versions in `core/tests/protractor-mobile.conf.js`, else the whole code remains the same?
> 
> EDIT: Also, can you please provide 5 screenshots of 5 straight successful runs on CI for the mobile version of contributor dashboard suite.

I have run the workflow 5 times on CI [here](https://github.com/oppia/oppia/runs/7079976093?check_suite_focus=true). (the mobile tests were successful)

**NOTE:** On the 3rd attempt, the Blog Dashboard Test failed (maybe a flake?).

![Screenshot from 2022-06-28 06-32-55](https://user-images.githubusercontent.com/73544247/176202180-fd8c9610-5ea0-430b-95d2-07b99e4b2fb3.png)